packages/client-sdk-web/src/internal/leaderboard-client.ts

```diff
@@ -71,15 +141,17 @@ export class LeaderboardDataClient<
     offset?: bigint | number,
     count?: bigint | number
   ): Promise<LeaderboardFetch.Response> {
+    const offsetValue = offset === undefined ? 0n : BigInt(offset);
+    const countValue = count === undefined ? 8192n : BigInt(count);
```

Comment:
i thought the protos for this were changing so we can explicitly specify undefined? maybe that is coming in a follow-up PR?

Reply:
I'm not sure. The explicit Unbounded values were for specifying the ScoreRange. 

From the [cache client protos](https://github.com/momentohq/client-protos/blob/main/proto/cacheclient.proto#L804), it seems the offset and count fields use default values rather than undefined.